modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -182,6 +182,24 @@ supports up to `64` threads. If this field is empty, the Ingress Controller uses
 * `tlsInspectDelay` specifies how long the router can hold data to find a matching route. Setting this value too short can cause the router to fall back to the default certificate for edge-terminated or reencrypted routes, even when using a better matched certificate. If unset, the default inspect delay is `5s`.
 
 * `tunnelTimeout` specifies how long a tunnel connection, including websockets, remains open while the tunnel is idle. If unset, the default timeout is `1h`.
+
+|`logEmptyRequests`
+|`logEmptyRequests` specifies how connections on which no request is received and logged. These empty requests come from load balancer health probes or web browser speculative connections, preconnect, and logging these requests can be undesirable. However, these requests can be caused by network errors, in which case logging empty requests can be useful for diagnosing the errors. These requests can be caused by port scans, and logging empty requests can aid in detecting intrusion attempts. Allowed values for this field are `Log` and `Ignore`. The default value is `Log`.
+
+The `LoggingPolicy` type accepts either one of two values:
+
+* `Log`: Setting this value to `Log` indicates that an event should be logged.
+* `Ignore`: Setting this value to `Ignore` sets the `dontlognull` option in the HAproxy configuration.
+
+|`HTTPEmptyRequestsPolicy`
+|`HTTPEmptyRequestsPolicy` describes how HTTP connections are handled if the connection times out before a request is received. Allowed values for this field are `Respond` and `Ignore`. The default value is `Respond`.
+
+The `HTTPEmptyRequestsPolicy` type accepts either one of two values:
+
+* `Respond`: If the field is set to `Respond`, the Ingress Controller sends an HTTP `400` or `408` response, logs the connection, if access logging is enabled, and counts the connection in the appropriate metrics.
```

Comment:
```suggestion
* `Respond`: If the field is set to `Respond`, the Ingress Controller sends an HTTP `400` or `408` response, logs the connection if access logging is enabled, and counts the connection in the appropriate metrics.
```

You don't need a comma after between "connection" and "if access" (if I'm understanding this sentence correctly).